src/policy/policy.cpp

```diff
@@ -183,3 +183,9 @@ int32_t Policy::blockSizeAcceptLimit()
         LogPrintf("BlockSize set to extremely low value (%d bytes), this may cause failures.\n", limit);
     return limit;
 }
+
+int64_t Policy::blockSigOpAcceptLimit(int32_t nBlockSize)
+{
+    uint64_t nBlockSizeMb = 1 + (nBlockSize - 1) / 1000000;
```

Comment:
No ceil works on floating point. You don't want floating point in consensus code, as they tend to be non deterministic. Notably on x86 rounding is controlled by a flag register that isn't saved/restored as part of the ABI, even when doing round trip to the kernel. There are war story about device driver changing the floating point rounding mode and causing application to misbehave.

For clarity, please put () around the division.